src/main/java/org/opensearch/security/support/WildcardMatcher.java

```diff
@@ -255,18 +233,6 @@ public boolean matchAny(String... candidates) {
         return matchAny(Arrays.stream(candidates));
     }
 
-    public boolean matchAll(Stream<String> candidates) {
-        return candidates.allMatch(this);
-    }
-
-    public boolean matchAll(Collection<String> candidates) {
-        return matchAll(candidates.stream());
-    }
-
-    public boolean matchAll(String[] candidates) {
-        return matchAll(Arrays.stream(candidates));
-    }
```

Comment:
@nibix do you remember why was this removed? AWS had a dependency on this method (as a patch) which now seems broken. is it okay to bring it back as it is?

Reply:
Regarding the commented code in https://github.com/opensearch-project/security/blob/2.19/src/main/java/org/opensearch/security/securityconf/ConfigModelV7.java#L566-L571 

My guess would be that this code is this way since the inception of OpenSearch (or OpenDistro). It stems from old ES 6 times, when there were still doc types. It is totally safe to remove that code. I guess there are a couple of more instances like this.

Reply:
> Yeah some more commented too. They just add confusion during audits.

I just checked a bit of the code: Good news is that we could remove quite a lot of that with the optimized privilege evaluation PR.